src/sentry/issues/grouptype.py

```diff
@@ -560,12 +569,14 @@ class MonitorIncidentType(GroupType):
 class MonitorCheckInTimeout(MonitorIncidentType):
     # This is deprecated, only kept around for it's type_id
     type_id = 4002
+    slug = "monitor_check_in_timeout"
 
 
 @dataclass(frozen=True)
 class MonitorCheckInMissed(MonitorIncidentType):
     # This is deprecated, only kept around for it's type_id
     type_id = 4003
+    slug = "monitor_check_in_missed"
```

Comment:
added slugs in order to not have collisions with the parent slug upon registration.

Reply:
I'm not sure if we should do this... I don't know if the slugs are stored anywhere that this would break things

Reply:
@wedamija is using the `type_id` better?

Reply:
edit: no registry means i don't need to add this